pyside6_overlay/ui/components/status_bar_ui.py

```
@@ -0,0 +1,50 @@
+
+"""
+Status Bar UI Component - Manages status information display
+"""
+
+from PySide6.QtWidgets import QWidget, QHBoxLayout
```

Ruff (0.11.9)

`PySide6.QtWidgets.QWidget` imported but unused

Remove unused import: `PySide6.QtWidgets.QWidget`

(F401)